man/org.freedesktop.systemd1.xml

```diff
@@ -1665,13 +1665,21 @@ node /org/freedesktop/systemd1 {
           </ulink>.</para>
 
           <xi:include href="version-info.xml" xpointer="v252"/></listitem>
+        </varlistentry>
 
+        <varlistentry>
+          <term><literal>var-run-bad</literal></term>
+
+          <listitem><para><filename>/run/</filename> does not exist or <filename>/var/run</filename> is not a
```

Comment:
suffix dir paths with / please

Reply:
So we're referring to the symlink here, not the actual directory. I think / suffix would be somehow misleading?

Reply:
I agree with @YHNdnzj here. At least, the sentence is not introduced by this PR. Let's suffix them later if @poettering really wants.